packages/react/src/Row/Row.tsx

```diff
@@ -6,24 +6,43 @@
 import clsx from 'clsx'
 import { forwardRef } from 'react'
 import type { HTMLAttributes, PropsWithChildren } from 'react'
+import type { CrossAlign, MainAlign } from '../common/layout'
 
 export const rowGapSizes: Array<string> = ['extra-small', 'small', 'medium', 'large', 'extra-large']
 
-type RowTag = 'article' | 'div' | 'section'
 type RowGap = (typeof rowGapSizes)[number]
+type RowTag = 'article' | 'div' | 'section'
 
 export type RowProps = {
+  /** The horizontal alignment of the items in the row. */
```

Comment:
Maybe use `inline` here instead of `horizontal`, to educate our users?

Reply:
No, too difficult. Maybe jargon, even 😉 Horizontal is far more natural – even I still need to translate ‘block’ and ‘inline’ to physical directions, let alone non-devs.